packages/conf-brotli/conf-brotli.0.0.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "fxfactorial"
+authors: "Edgar Aroutiounian"
```

Comment:
Could you please specify the authors of the software here, i.e. "The Brotli Authors" (as per the license file of brotli).